### PR TITLE
Making Degreed integration configuration fields required

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
Setting all configuration fields to required. Turns out, the Degreed factory already accounts for all fields so no changes required.

I took a look at staging and saw that there were no configurations. I'm assuming that any configuration on prod has all the fields, but perhaps we should check? As to make sure nothing breaks when the required fields go live.

**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [ ] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/edx/edx-enterprise/actions), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

File: integrated_channels/degreed/migrations/0009_auto_20210119_1546.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='key',
+            field=models.CharField(help_text='The API Client ID provided to edX by the enterprise customer to be used to make API calls to Degreed on behalf of the customer.', max_length=255, verbose_name='API Client ID'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='secret',
+            field=models.CharField(help_text='The API Client Secret provided to edX by the enterprise customer to be used to make API calls to Degreed on behalf of the customer.', max_length=255, verbose_name='API Client Secret'),
```
Comment (taliaedX) [inline integrated_channels/degreed/migrations/0009_auto_20210119_1546.py:71]: part of me is thinking "let's make the field less of a scroll-y boy", but also ehhh.

**Review by taliaedX — APPROVED** [review]
One slight nit, but also I don't care too strongly.
One thing to do - if you haven't already done it - is to add a degreed config locally BEFORE this migration is applied (go ahead and fill in all fields) and verify it doesn't throw an error for some reason (it really shouldn't, but JUST IN CASE).

Also to your PR description around verifying in Prod: I did verify all configs have all fields filled in on Prod prior to making the ticket, however, if you aren't sure how to verify and would love to see it with your own eyes, let me know! I can show you my process.

**Comment by alex-sheehan-edx** [comment]
> add a degreed config locally BEFORE this migration is applied (go ahead and fill in all fields) and verify it doesn't throw an error for some reason

Goooooood idea... I've just already applied my migrations. Gonna have to remember how to step back

> I did verify all configs have all fields filled in on Prod prior to making the ticket

Nice! 

> however, if you aren't sure how to verify and would love to see it with your own eyes, let me know

This would be accessing Django Admin on prod? Or I guess checking the DB? Sure! Let's find a time to go through it together.

File: integrated_channels/degreed/migrations/0009_auto_20210119_1546.py (hunk 1 above)
Comment (alex-sheehan-edx) [reply integrated_channels/degreed/migrations/0009_auto_20210119_1546.py:71]: do you mean the length of the line? I can happily move it to multiple lines.

Comment (alex-sheehan-edx) [reply integrated_channels/degreed/migrations/0009_auto_20210119_1546.py]: done!

**Pull request merged**